learn/src/main/java/org/navgurukul/learn/ui/learn/ListSpacingDecoration.kt

```diff
@@ -0,0 +1,166 @@
+package org.navgurukul.learn.ui.learn.adapter
+
+import android.content.Context
+import android.graphics.Rect
+import android.view.View
+import androidx.annotation.DimenRes
+import androidx.recyclerview.widget.*
+import androidx.recyclerview.widget.RecyclerView.ItemDecoration
+
+
+class ListSpacingDecoration : ItemDecoration {
```

Comment:
This should be moved to common module if something doesn't exist. AFAIK something like this already exists